### DISCUSSION TITLE
Auto add a space when auto complete a python keyword like import, def, for, if and so on.

### DISCUSSION THREAD

**Discussion by shenmishajing** [microsoft/pylance-release#3862] [Enhancement]
We use almost all of python keywords (except break, continue, None, True and False) followed by a space, but in the auto complete of vscode-python, when we accept a suggestion like import, def, for, if and other keywords, vscode do not auto add the space for us. Therefore, we have to add the space manually, which will be a little bit annoying. Can we change this to auto add a space after all keywords except break, continue, None, True and False?

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.
